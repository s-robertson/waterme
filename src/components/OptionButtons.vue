<template>
  <div class="c-option-buttons">
    <slot />
  </div>
</template>

<script>
export default {
  name: "OptionButtons"
};
</script>

<style lang="scss">
.c-option-buttons {
  display: flex;
  align-items: center;

  * + * {
    margin-left: 0.5rem;
  }
}
</style>
