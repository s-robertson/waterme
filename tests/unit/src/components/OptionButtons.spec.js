import { createLocalVue, shallowMount } from "@vue/test-utils";
import OptionButtons from "@/components/OptionButtons";

const localVue = createLocalVue();

describe("OptionButtons component", () => {
  it("Should render with slotted children", () => {
    const wrapper = shallowMount(OptionButtons, {
      localVue,
      slots: {
        default: "hello there"
      }
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
