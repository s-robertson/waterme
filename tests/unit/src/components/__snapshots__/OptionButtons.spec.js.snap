// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OptionButtons component Should render with slotted children 1`] = `<div class="c-option-buttons">hello there</div>`;
